### DISCUSSION TITLE
Emoji 15.1 Beta

### DISCUSSION THREAD

**Discussion by C1710** [C1710/blobmoji#134] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/C1710/blobmoji/releases/tag/v15.1-beta1'>Emoji 15.1 Beta</a>.</em>

**Comment by DavidBerdik** [comment]
The Magisk module for this release is available here: https://github.com/DavidBerdik/blobmoji/releases/tag/15.1-beta-r1

**Comment by idontveapc** [comment]
im not using it as the system font, and still emojis with `variation selector-16` are showed from the system and rest from the blob emojis.

**Comment by ikkqqstudio** [comment]
plz zfont 3 rootless device support
i need to apply in zfont 3
zfont 3 is rootless font changing
but i try this and it space too much and number nocilped can you add rootless zfont 3

**Comment by HaNerW10** [comment]
Hello, i need the 😛😜😝 to be the Android 6.0-6.0.1 ones. I really miss the old ones which were looking neutral and i really like the old design of tongue emojis. So could you please update it to the old ones? Please!

**Comment by imaddictedtoemojis** [comment]
Emoji 16 blobmoji emojis! You can use this to make Emoji 16.0 Blobmojis.
Tell me what should be changed!
![emoji_u1fae9](https://github.com/user-attachments/assets/aad1f714-dd92-4c0b-a925-ae615eb61ed2)
![splatter blobs (2)](https://github.com/user-attachments/assets/e46d0007-689a-4c3f-a684-cd8cb2da1107)
![shovel blobs](https://github.com/user-attachments/assets/467589e8-7a16-470c-9e05-2271c8c81255)
![fingerprint blobs (1)](https://github.com/user-attachments/assets/fed99172-103c-41c6-920c-69c7192895b2)
![harp blobs (1)](https://github.com/user-attachments/assets/9cbbc052-3f61-43a5-a1b7-ce05c7d27f1e)
![leafless tree blobs](https://github.com/user-attachments/assets/4d0d8d07-a9d7-4657-9d85-70880b9a8269)
![root vegetables emoji](https://github.com/user-attachments/assets/4e6db264-5201-4873-9dca-de5c6877b1f6)

**Comment by hgggy7777** [comment]
I'm imaddictedtoemojis alt and check issues github.com/C1710/blobmoji/issues/163 Unicode 16, github.com/C1710/blobmoji/issues/164 for Unicode 17
